site/docs/tools/deltalake/spark.md

```diff
@@ -11,13 +11,13 @@ In pyspark this would look like
 ``` python
 SparkSession.builder
     .config('spark.jars.packages',
-            'org.projectnessie:nessie-deltalake:{{ versions.java }}')
+            'org.projectnessie.nessie-integrations:nessie-deltalake:{{ versions.java }}')
     ... rest of spark config
     .getOrCreate()
 ```
 
 In order to utilize the additional SQL grammar from the Nessie Spark SQL Extensions make sure to
-also include `org.projectnessie:nessie-spark-3.2-extensions:{{ versions.java }}` and
+also include `org.projectnessie.integrations:nessie-spark-3.2-extensions:{{ versions.java }}` and
```

Comment:
Should this be `nessie-integrations` instead of just `integrations`?

source: https://repo1.maven.org/maven2/org/projectnessie/nessie-spark-extensions-3.3_2.12/0.51.1/nessie-spark-extensions-3.3_2.12-0.51.1.pom

I can open a PR. If you confirm this.

Reply:
cc: @snazy, @adutra

Reply:
I am sure now that it is a typo. fixed via https://github.com/projectnessie/nessie/pull/6259